pkg/agent/controller/networkpolicy/reject.go

```diff
@@ -150,7 +150,19 @@ func (c *Controller) rejectRequest(pktIn *ofctrl.PacketIn) error {
 		srcMAC = sIface.MAC.String()
 		dstMAC = dIface.MAC.String()
 	}
-	inPort, outPort := getRejectOFPorts(packetOutType, sIface, dIface)
+	getInterfaceOFportByType := func(ifaceType interfacestore.InterfaceType) uint32 {
```

Comment:
Add this func to ifaceStore?

Reply:
Think deeper, maybe a better way is passing the gateway port and tunnel port from NodeConfig when constructing network policy controller. Then the inner function is not needed, and all the consumers for these two ports are actually from NodeConfig. What do you think? @gran-vmv

Reply:
Sounds good.